src/Kunstmaan/MediaBundle/Helper/MediaManager.php

```diff
@@ -160,4 +180,22 @@ public function getFolderAddActions()
 
         return $result;
     }
+
+    public function isExtensionAllowed(File $file): bool
+    {
+        $mimeType = $this->mimeTypes->guessMimeType($file->getRealPath());
+        $extension = $this->mimeTypes->getExtensions($mimeType)[0] ?? 'not_allowed';
```

Comment:
```suggestion
        $extensions = $this->mimeTypes->getExtensions($mimeType);
        $extension = array_shift($extensions);
```

This is maybe a better approach, as the value return by `$this->mimeTypes->getExtensions()` will always be an array. `array_shift` will return `null` if the array is empty but that won't break the `in_array` checks.